.github/workflows/fetch-coordinape-data-daily.yml

```diff
@@ -0,0 +1,84 @@
+name: Fetch Coordinape Data (Daily Cron Job)
+
+on:
+  # A new interval every day (midnight)
+  schedule:
+    - cron: 0 0 * * * # 00:00 UTC
+  push:
+    branches:
+      # Allows us to test this workflow
+      - "fetch-coordinape-data-trigger-*"
+
+jobs:
+  coordinape_users:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: development-guild/contributions/
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Pull changes
+        run: |
+          git fetch
+          git pull
+
+      - name: Use Node.js 18.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+          cache: npm
+          cache-dependency-path: development-guild/contributions/package-lock.json
+      
+      - name: Install Node.js Packages 🔧
+        run: npm install
+
+        # Fetch Coordinape users and store them in a CSV file 🧮
+      - run: node fetch-users.js
+      
+      - name: Commit Changes
+        run: |
+          git config user.name 'NationCred bot'
+          git config user.email 'nationcred.bot@nation3.org'
+          git add output/*
+          git commit --allow-empty -m 'fetch coordinape users'
+      
+      - name: Push Changes
+        run: git push
+
+  coordinape_contributions:
+    needs: coordinape_users
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: development-guild/contributions/
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Pull changes
+        run: |
+          git fetch
+          git pull
+
+      - name: Use Node.js 18.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+          cache: npm
+          cache-dependency-path: development-guild/contributions/package-lock.json
+      
+      - name: Install Node.js Packages 🔧
+        run: npm install
+
+        # Fetch Coordinape contributions and store them in a CSV file 🧮
+      - run: node fetch-contributions.js
+      
+      - name: Commit Changes
+        run: |
+          git config user.name 'NationCred bot'
+          git config user.email 'nationcred.bot@nation3.org'
+          git add output/*
+          git commit --allow-empty -m 'fetch coordinape contributions'
+      
+      - name: Push Changes
+        run: git push
```

Comment:
I haven't bee part of this, but it is cool, should this be two actions *or* should one trigger the other e.g fetch users triggers fetch contributions?

Reply:
@johnmark13 I ended up refactoring it so that there will be one job per guild:  https://github.com/nation3/citizen-rewards/pull/6/commits/c403cab910b333cd04f7ff065221be6fb15498e8#diff-fb31991cd2b5b54c3c9ccb08ca74326ea4ab6a48c4f4243d526218b856812773